OpenNefia.Content/CharaMake/CharaMakeAttributeRerollLayer.cs

```diff
@@ -129,8 +129,11 @@ public override void Initialize(CharaMakeData args)
         {
             // gets called when the ui is loaded, so this could potentially run multiple times 
             //      if the user backtracks in creation
-            if (IsInitialized) 
+            if (IsInitialized)
+            {
+                Reroll(false);
```

Comment:
This will not reset locked items/attribute numbers if you go back, change races/classes and go forward again.

Reply:
behaviour is fixed in 11931b4